Assets/WalletConnectUnity/Scripts/WalletConnectSharp.Unity/UI/WalletConnectQRImage.cs

```diff
@@ -26,23 +37,47 @@ void OnEnable()
             return;
         }
         
-        walletConnect.ConnectionStarted += WalletConnectOnConnectionStarted;
+        // Only register the WalletConnectOnConnectionStarted handler once
+        if (!registeredOnConnectionStartEvent)
+        {
+            registeredOnConnectionStartEvent = true;
+            walletConnect.ConnectionStarted += WalletConnectOnConnectionStarted;
```

Comment:
I guess we should be using the new `walletConnect.NewSessionStarted` event handler to only trigger a QRCode update when a new session is created. 

Having the bool flag check here is still needed so we dont subscribe more than once

Reply:
Actually never mind, we want the QR Code to generate when we connect to the bridge, not after the session gets created/connected